src/test/groovy/pl/allegro/tech/discovery/consul/recipes/watch/health/HealthServiceInstancesWatcherIntTest.groovy

```diff
@@ -34,13 +35,19 @@ class HealthServiceInstancesWatcherIntTest extends Specification {
     private EndpointWatcher<Services> healthServiceInstancesWatcher = recipes.healthServiceInstancesWatcher("my-service",
             recipes.consulWatcher(Executors.newFixedThreadPool(1))
                     .withBackoff(100, 1000)
-                    .build())
+                    .build()) as EndpointWatcher<Services>
+
+    def cleanup() {
+        consulCluster.findAllServices("dc1", "node1-dc1").getValue().forEach { key, val ->
```

Comment:
I would put this to separate method with clear name (like "deregisterServices"), and extract this strings to consts

Reply:
i though about it also, so if we both do I changed it as you pointed